llvm/lib/TableGen/Record.cpp

```diff
@@ -92,10 +92,39 @@ struct RecordKeeperImpl {
 
   unsigned AnonCounter;
   unsigned LastRecordID;
+
+  void dumpAllocationStats(raw_ostream &OS) const;
 };
 } // namespace detail
 } // namespace llvm
 
+void detail::RecordKeeperImpl::dumpAllocationStats(raw_ostream &OS) const {
+  // Dump memory allocation related stats.
+  OS << "TheArgumentInitPool size = " << TheArgumentInitPool.size() << "\n";
```

Comment:
```suggestion
  OS << "TheArgumentInitPool size = " << TheArgumentInitPool.size() << '\n';
```
Single quotes for single characters

Reply:
Done.